source/fundamentals/collations.txt

```diff
@@ -0,0 +1,280 @@
+==========
+Collations
+==========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+This guide shows you how to use **collations**, a set of sorting rules, to
+run operations using string ordering for specific languages and locales (a
+community or region that shares common language idioms).
+
+MongoDB sorts strings using *binary collation* by default. This collation
+method uses the `ASCII standard <https://en.wikipedia.org/wiki/ASCII>`_
+character values to compare and order strings. Languages and locales
+have specific character ordering conventions that differ from the ASCII
+standard.
+
+For example, in Canadian French, the right-most accented character determines
+the ordering for strings when the other characters are the same. Consider the
+following French words: **cote**, **coté**, **côte**, and **côté**.
+
+The order in which they would appear using the default *binary
+collation* is:
+
+::
+
+   cote
+   coté
+   côte
+   côté
+
+If we apply the Canadian French collation, the order is:
+
+::
+
+   cote
+   côte
+   coté
+   côté
+
+.. note::
+
+   Collations are available in MongoDB 3.4 and later.
+
+-----
+Usage
+-----
+
+You can specify a collation when you create a **new collection** or
+**new index**. You can also specify a collation for :doc:`CRUD operations
+</fundamentals/crud>` and aggregations.
+
+When you create a **new collection** with a collation, you define the default
```

Comment:
Not sure how I feel about bolding "new collection". I feel like we might be better off leaving the bold out since "new collection" and "new index" aren't really concepts. This almost makes me feel like I'm reading java (`new Collection()`).

Reply:
This was an editorial choice I made to emphasize that it must be a "new" collection or index and not an existing one. I'll take away the formatting.